Use case insensitive like for pgsql

circulon opened the pull request.

circulon committed:
Use case insensitive like for pgsql

josephmancuso commented:
@circulon These tests also need to be fixed. Same thing just changing LIKE to ILIKE

```
FAILED tests/sqlite/builder/test_sqlite_query_builder.py::SQLiteQueryBuilderTest::test_where_like
196
FAILED tests/sqlite/builder/test_sqlite_query_builder.py::SQLiteQueryBuilderTest::test_where_not_like
197
FAILED tests/sqlite/grammar/test_sqlite_select_grammar.py::TestSQLiteGrammar::test_where_like
198
FAILED tests/sqlite/grammar/test_sqlite_select_grammar.py::TestSQLiteGrammar::test_where_not_like
```

circulon committed:
Fixed incorrect location of grammar change

circulon commented:
@josephmancuso 

Oops totally updated the wrong grammar file.
Reverted the Sqlite grammar as there is no 'ILIKE' operater 
Added the postgres grammar change as I should have origanally.

circulon committed:
Updated Tests

circulon committed:
Updated Grammar Tests

josephmancuso commented:
Looks Great!
